packages/dd-trace/src/appsec/index.js

```diff
@@ -121,16 +121,16 @@ function incomingHttpEndTranslator ({ req, res }) {
   }
 
   // TODO: temporary express instrumentation, will use express plugin later
-  if (req.params && typeof req.params === 'object') {
+  if (req.params !== null && typeof req.params === 'object') {
```

Comment:
actually now that i think of it, why change it to explicit null check ?
most of the time the value will either be undefined, so that would require the execution of typeof, or it would be an object, basically almost never null. At last with the truthy check we skip the typeof check for undefined yknow ?

Reply:
That's also why I left these untouched in the first version of the PR. However, I do also like consistency, and while I'm here fixing the actual issues as well, I might as well make it all constant. But I'm happy to revert that change if you think it's better to not touch this

Reply:
eh whatever it's microoptimization anyway :+1: